src/test/java/org/gridsuite/study/server/ShortCircuitTest.java

```diff
@@ -549,7 +549,7 @@ private StudyEntity insertDummyStudy(UUID networkUuid, UUID caseUuid) {
                 .hvdcAcEmulation(true)
                 .build();
         ShortCircuitParametersEntity defaultShortCircuitParametersEntity = ShortCircuitService.toEntity(ShortCircuitService.getDefaultShortCircuitParameters());
-        StudyEntity studyEntity = TestUtils.createDummyStudy(networkUuid, caseUuid, "", "defaultLoadflowProvider", defaultLoadflowParametersEntity, defaultShortCircuitParametersEntity, null);
+        StudyEntity studyEntity = TestUtils.createDummyStudy(networkUuid, caseUuid, "", "defaultLoadflowProvider", defaultLoadflowParametersEntity, defaultShortCircuitParametersEntity, null, null);
```

Comment:
why not passing a defaultSensitivityAnalysisParametersEntity instead of null ?

Reply:
done